src/styles/sidebar-tutorial.scss

```diff
@@ -44,10 +54,45 @@
   color: $gray-light;
 }
 
+.sidebar-tutorial__list-item--is-theme-clean {
+  font-size: 14px;
+}
+
+.sidebar-tutorial__list-item--is-theme-clean::before {
+  content: counter(sidebar-tutorial__list) ".";
+  display: table-cell; /* aha! */
+  text-align: right;
+  padding-right: .3em;
+  color: $gray-dark;
+}
+
 .sidebar-tutorial__list-item-content {
   margin-top: 1em; /* Put vertical space in-between list items, equal to the
                       space between normal paragraphs.
                       Note: This also puts the same amount of space above the
                       first list item (i.e. between the list and whatever's
                       above it). */
 }
+
+.sidebar-tutorial__list-item-annotate {
+  background-color: #b9d9fa;
+}
+
+.sidebar-tutorial__list-item-new-note-btn {
+  background-color: #626262;
+  border: none;
+  border-radius: 3px;
+  color: #fff;
+  display: -ms-flexbox;
```

Comment:
CSS property values should be unprefixed (ie. `display: flex`). `-ms-flexbox` is a legacy value for IE 10/11. Our CSS preprocessor will add prefixes for the browsers we support. The exceptions are a few rare properties that only exist in one browser.